draft-ietf-scone-protocol.md

```diff
@@ -317,18 +317,19 @@ A Rate Signal is a 7-bit unsigned integer (0-127). The high six bits are the
 Rate Signal High Bits, and the least significant bit is the most significant
 bit of the Version field.
 
-When sent by a QUIC endpoint, the Rate Signal is set to 127.  Receiving a value
-of 127 indicates that throughput advice is unknown, either because network
-elements on the path are not providing advice or they do not support SCONE. All
-other values (0 through 126) represent the ceiling of rates advised by the
-network element(s) on the path.
+When sent by a QUIC endpoint, the Rate Signal MUST be set to 127.  Receiving a
+value of 127 indicates that throughput advice is unknown or unchanged, either
+because network elements on the path are not providing new or updated advice or
+they do not support SCONE.  126 explicitly indicates there is no advised
+limit. All other values (0 through 125) represent the ceiling of rates advised
+by the network element(s) on the path.
 
```

Comment:
Not sure about this MUST. I understand that the right thing happens if clients follow that advice, but how exactly is that enforced? Network elements MUST absolutely be ready to receive SCONE packets that were somehow marked by another Network Element on the same path. I would be concerned if implementers of network elements treated incoming SCONE packets with a Rate Signal different from 127 as an error.
I think that we should write "SHOULD" instead. As in: 

When sent by a QUIC endpoint, the Rate Signal SHOULD be set to 127. Other values MAY be used, for example for testing purposes, but setting the value to 127 ensure proper
processing by the advice-receiving endpoint.